app/controllers/api/v2/hosts_controller.rb

```diff
@@ -125,6 +125,7 @@ def show
 
       api :POST, "/hosts/", N_("Create a host")
       param_group :host, :as => :create
+      param :overwrite, :bool, :desc => N_("true by default")
```

Comment:
I don't see the parameter being used anywhere, am I missing something?
Also it would be very helpful to have bit more descriptive description for it.

Reply:
There it is
https://github.com/theforeman/foreman/blob/f5a3dd2dedb569a0e6c5e8bb311c1da75906fc4f/app/controllers/concerns/foreman/controller/parameters/host_base.rb#L18
Or am i wrong ?

in `hammer host create -h` the parameter does not have descriptive description 😃 
```
--overwrite OVERWRITE      One of true/false, yes/no, 1/0.
                                   Default: "true"
```
So I thought it wasn't that important

Reply:
We need to nest the `overwrite` under the `:host`. @ofedoren can you help us how to go about it to add overwrite to the `param_group`, but only for create action?

Reply:
Sorry for the late reply. @maccelf, you can re-open the param group:

```ruby
      api :POST, "/hosts/", N_("Create a host")
      param_group :host, :as => :create
      param :host, Hash do
        param :overwrite, :bool, :desc => N_("Overwrite existing host true by default")
      end
```

Reply:
Thanks @ofedoren